project/src/analysis/rdf.py

```diff
@@ -0,0 +1,35 @@
+"""Facilitate the calculation of the RDF from simulation data."""
+
+import freud
+import gsd
+import gsd.hoomd
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+def gsd_rdf(job):
+    """Compute the RDF given a GSD file."""
+    gsdfile = job.fn("trajectory.gsd")
+
+    # TODO check that these are reasonable values
+    frames = 10
+    bins = 50
+    r_min = 0.5
+    r_max = 2.5
```

Comment:
Shall we change r_max to the (minimum (list of side length in each prod traj frame))/2-0.01 ?

Reply:
As the longest possible r_max, that is a good place to start! Depending on how big our systems get, we may want to go smaller, but I agree that we should have this as the default.

Reply:
I think freud complains when r_max > L/2. But we can confirm that once we start the analysis.

Reply:
Yes, you are correct. I have updated this code to use a default r_max based on box lengths
(minimum box length * 0.45)